centreon/packages/ui/src/Form/CollapsibleGroup.tsx

```diff
@@ -95,7 +95,11 @@ const CollapsibleGroup = ({
         <ContainerComponent>
           {isCollapsible && <CollapseIcon />}
           <div className={classes.groupTitleIcon}>
-            <Typography className="groupText" variant="h5">
+            <Typography
+              className="groupText"
+              variant="h5"
+              {...group?.titleAttributes}
+            >
```

Comment:
it's fine for the variant property to be set twice? Additionally, regarding the className value "groupText," if it does not correspond to any existing class, it may be advisable to remove it. Instead, consider utilizing the className prop to override the default className for consistency with how we have handled other components.

Reply:
To override it, it makes no sense to override all the styles of variant h5 (fontFamily, fontWeight, fontSize, lineHeight) using className, as you suggest. Otherwise, I can simply use the convenient variant. Additionally, for the titleAttributes, it allows you to pass all the props of typography without the need to override the style on Centreon web  :)